lib/bundle/locker.rb

```diff
@@ -20,25 +22,58 @@ def lockfile(global: false, file: nil)
       lock_file_path
     end
 
-    def write_lockfile?(global: false, file: nil, no_lock: false)
+    def write_lockfile?(path: nil, no_lock: false)
       return false if no_lock
       return false if ENV["HOMEBREW_BUNDLE_NO_LOCK"]
 
       # handle the /dev/stdin and /dev/stdout cases
-      return false if lockfile(global: global, file: file).parent.to_s == "/dev"
+      return false if path.nil? || path.parent.to_s == "/dev"
+
+      true
+    end
 
+    def write_lockfile!(lockfile, lock_data)
```

Comment:
Personally I don't think it makes things easier to follow to refactor methods called only once into other methods.

Reply:
It feel like it improves the readability and testability, although I ran out of time while working on this to expand the tests to cover these smaller units. I did it initially to reason better about the steps taken: I found myself adding comments to explain the steps, and so many times when I write comments, I realize they could be method calls. I also like to keep Ruby methods short. I think Rubocop's default is like 10 lines; I'm not sure what [the brew rubocop](https://github.com/Homebrew/brew/blob/master/Library/.rubocop.yml) ends up resolving to.